FabricExample/package.json

```diff
@@ -39,6 +40,7 @@
     "@react-native/metro-config": "0.74.83",
     "@react-native/typescript-config": "0.74.83",
     "@types/react": "^18.2.72",
+    "@types/react-native-restart": "^0.0.0",
```

Comment:
Weird version 🤔

Reply:
You're right, I copied dependencies from the removed package.json to ensure nothing's missing but this types package looks strange. The [react-native-restart ](https://www.npmjs.com/package/react-native-restart) package offers should be ts ready by itself
[refact:removed unused types](https://github.com/software-mansion/react-native-screens/pull/2174/commits/bc6e06a6b444e219ff029d18ef12441391995686)